src/plugins/discover/public/application/angular/discover.js

```diff
@@ -990,6 +992,9 @@ function discoverController($element, $route, $scope, $timeout, $window, Promise
       operation,
       $scope.indexPattern.id
     );
+    if (trackUiMetric) {
+      trackUiMetric(METRIC_TYPE.CLICK, 'filter_added');
```

Comment:
Is there a reference to Discover recorded when it's added like this? or else I think it should be `discover_filter_added`

Reply:
Metric name is always prefixed with the plugin name:
```
{
  "type":"ui-metric",
  "id":"discover:filter_added",
  "attributes":{"count":2},
...
}
```

It's just in the case of the shared components, like the search bar, the prefix is `data_plugin`, so I needed to add the name of the app explicitly: https://github.com/elastic/kibana/pull/82709/files#diff-b370985f7060e1ee97121c246b0ac878bf5b1dee2271e0593a983f893141109eR136

In that case, the final metric name is: `data_plugin:discover:filter_added`

Reply:
good to know, thx for the context 👍